CONTRIBUTING.md

```diff
@@ -97,6 +97,7 @@ source files, respectively.
 **NOTE**: We use specific versions of formatting tools to ensure consistency across the project. The required versions are:
 - clang-format version **15.0**, which can be installed with the command: `python -m pip install clang-format==15.0.7`.
 - cmake-format version **0.6**, which can be installed with the command: `python -m pip install cmake-format==0.6.13`.
+- black version **24.3**, which can be installed with the command: `python -m pip install black==24.3.0`.
```

Comment:
hmm.. this section is about `specific required versions` of these tools. We don't require a specific version of `black` (it has rather good compatibility between versions (in comparison to clang-format, which may change too much between versions).

So... I would not add this info here...?


//
General information how to install black (including suggested version) is in the top-level Readme - https://github.com/oneapi-src/unified-memory-framework?tab=readme-ov-file#build

Reply:
Do you wish to leave the suggested version number? Simply `python -m pip install black` should suffice.

I have removed the version number in the updated version.

Reply:
Or maybe you mean that you don't want to leave the info about black at all?

Reply:
I mean this change does not introduce anything new in here, I believe. The same info is in the top-level README.

This section, useful for contributors, says `The required versions are: ...` and `black` does not have any specific version required (at least at the moment).

Reply:
ok, I see your point now. Then perhaps we could write something like this:

```
- black - no specific version required (can be installed with the command: `python -m pip install black==24.3.0)`.
```

Reply:
Done